sycl/include/CL/sycl/INTEL/esimd/detail/esimd_util.hpp

```diff
@@ -10,12 +10,19 @@
 
 #pragma once
 
+#include <CL/sycl/INTEL/esimd/detail/esimd_types.hpp>
 #include <CL/sycl/detail/type_traits.hpp>
 
-namespace __esimd {
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+namespace INTEL {
+namespace gpu {
+namespace detail {
 
-/// Constant in number of bytes.
-enum { BYTE = 1, WORD = 2, DWORD = 4, QWORD = 8, OWORD = 16, GRF = 32 };
+/// ESIMD intrinsic operand size in bytes.
+struct OperandSize {
```

Comment:
Maybe `enum class` ?

Reply:
OperandSize::member can be conveniently used as integer w/o static_cast, hence I did not add class